content/docs/getting_started/cleanup.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Cleanup"
+description: "Uninstall OSM and the bookstore applications"
+type: docs
+weight: 6
+---
+
+## Cleanup
+
+To cleanup all resources created for the demo, the OSM control plane, SMI resources, and the sample applications need to be deleted.
+
+To uninstall the sample applications and SMI resources, delete their namespaces with the following command:
+```bash
+kubectl delete ns bookbuyer bookthief bookstore bookwarehouse
+```
+
+To uninstall OSM, run
+```bash
+osm uninstall
```

Comment:
To avoid the cli prompt to confirm which mesh to uninstall we could specify `--mesh-name osm`.

Reply:
We don't want to assume what the mesh-name is. To avoid the prompt, use `osm uninstall -f`.

Reply:
I think letting users work through the prompt might actually be preferable here to show that it exists.

Reply:
I had assumed the mesh name was `osm` since in the install guide `content/docs/getting_started/install_osm.md` we use the default mesh name. If we are not going to make this assumption then I think it makes sense for the users to work through the prompt.